### PR TITLE
Add instructions on how to run the tutorial locally.

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+
+To run the tutorial locally, setup the environment with [conda](https://docs.conda.io/en/latest/miniconda.html) (or [mamba](https://mamba.readthedocs.io/en/latest/installation.html)):
+
+```console
+conda env create -f environment.yml
+```
+
+Then activate the environment with
+```console
+conda activate aiidalab-docker-stack
```
Comment (edoardob90) [inline README.md:16]: Shouldn't the environment name be `python-tutorial`? Or am I missing something?

Comment (yakutovicha) [reply README.md:16]: Yep, correct. Sorry for that 😄

Comment (yakutovicha) [reply README.md]: fixed in [28b16a8](https://github.com/empa-scientific-it/python-tutorial/pull/23/commits/28b16a8b8366dc15c7772074c4e2bcbf22f03ff6)

Comment (edoardob90) [reply README.md]: Great!

**Review by edoardob90 — APPROVED** [review]

**Pull request merged**